simularium_metrics_calculator/metrics_registry.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict
+
+from .calculators import (
+    AgentsCalculator,
+    NearestNeighborCalculator,
+    NumberOfAgentsCalculator,
+    TimesCalculator,
+)
+from .constants import METRIC_TYPE
+from .exceptions import MetricNotFoundError
+from .metric_info import MetricInfo
+
+metrics_list = [
+    MetricInfo(
+        uid=0,
+        display_name="Time",
+        metric_type=METRIC_TYPE.PER_TIME,
```

Comment:
There's something that feels funny to me about calling time a "PER_TIME" metric. I know you could technically plot time v time, so it is a time metric, but we're always going to be using it as an independent variable. I don't have a super helpful suggestion, but I just bump a little when I see the example of a time plot and you're just grabbing the `0`th item in the PER_TIME metrics and not just `t` or `time`. Basically I'm thinking that if we're grouping all the "per_time" metrics together for a "Y metric" on the front end, time shouldn't be in there, but with this current set up, I'm not sure how we'd tell the front end that other than knowing to look for "Time"

Reply:
I agree this is a little weird but not sure how to totally fix it. I added `MetricInfo.excluded_axes` like we talked about so you can specify that a metric is always independent (never on y-axis), so a user should never have to avoid a Time vs Time scenario on the front end